staging/traefik-forward-auth/values.yaml

```diff
@@ -81,7 +81,8 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
-env: {}
+env:
+  LOG_LEVEL: info
```

Comment:
should we default it to `trace` since that is what it was hardcoded to before?

Reply:
i think `trace` isn't an ideal default so i think we can keep this at info here.